packages/runtime/src/swagger/swagger.ts

```diff
@@ -217,7 +217,7 @@ export namespace Swagger {
 
   export interface Response3 {
     description: string;
-    content?: { [name: string]: Schema & Example };
+    content?: { [name: string]: { schema?: Schema } & Example };
```

Comment:
can schema really be optional? It's as optional everywehre else, therefore I did it here too.. but not sure if needed